ops/jenkins/global-pipeline-libraries/vars/awsSsm.groovy

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env groovy
+// awsSsm.groovy contains methods that wrap awscli ssm subcommands
+
+// Gets value from SSM parameter store based on parameter name.
+// If a SSM parameter does not exist, it will result in an exception.
+String getParameter(Map args = [:]) {
+    name = args.parameterName
+    awsRegion = args.awsRegion ?: 'us-east-1'
+
+    return sh(returnStdout: true, script: "aws ssm get-parameter --name ${name} --region ${awsRegion} --query 'Parameter.Value' --output text").trim()
+}
+
+// Stores a value in SSM parameter store based on the parameter name.  If a parameter doesn't exist in
+// parameter store, this function will store the value under the parameter name passed in and will
+// default the type stored as a 'String' unless otherwise passed in as a parameter type in the arguments.
+String putParameter(Map args = [:]) {
+    name = args.parameterName
+    value = args.parameterValue
+    type = args.parameterType ?: 'String'
+    shouldOverwrite = args.shouldOverwrite ? true : false
```

Comment:
Are you expecting a string for `args.shouldOverwrite`? This will always return `true` if the typing of `args.shouldOverwrite`  is a string (e.g. "false" == `true`). Otherwise you can just do `shouldOverwrite = args.shouldOverwrite`.

Reply:
`"false"` is definitely falsey in groovy.